### DISCUSSION TITLE
为什么会提示Cannot read property 'isCE' of null？

### DISCUSSION THREAD

**Discussion by 19774279** [element-plus/element-plus#14514] [Q&A]
我用vue create prj创建了一个vue3项目，然后运行npm install element-plus --save，按照官网介绍
1. 修改main.js
`import { createApp } from 'vue'
import ElementPlus from 'element-plus'
import 'element-plus/dist/index.css'
import App from './App.vue'

const app = createApp(App)

app.use(ElementPlus)
app.mount('#app')`
2. 再修改App.vue
`<template>
  <div>
    <el-button type="primary">primary</el-button>
  </div>
</template>`
3. 再用cnpm run serve
4. 访问http://localhost:8080，报错Uncaught TypeError: Cannot read property 'isCE' of null，为什么？

**Comment by xxholly32** [comment]
https://github.com/vuejs/core/issues/4344
